pilot/pkg/networking/plugin/plugin.go

```diff
@@ -64,10 +64,15 @@ type InputParams struct {
 	// Service is the service colocated with the listener (applies to sidecar).
 	// For outbound TCP listeners, it is the destination service.
 	Service *model.Service
+	// IP is the IP address on which the listener is listening
+	IP string
```

Comment:
These are easy to derive from the listener itself. Also this needs to be extendable to IPv6, multi IP binding, and UDS.

Reply:
IPv4 and IPv6 are covered as I am using net.IP/net.IPNet which automatically handles both v4/v6. UDS - we don't have a clear path for enabling UDS listeners yet. its a much larger change. So to keep things in scope, I stuck to IP.

removed IP though..